Frontend/CO.CDP.OrganisationApp/Pages/Organisation/SupplierToBuyerOrganisationType.cshtml

```diff
@@ -0,0 +1,143 @@
+                                </div>
+                            </div>
+                            <div class="govuk-radios__item">
+                            <input class="govuk-radios__input" id="@nameof(Model.BuyerOrganisationType)-3" name="BuyerOrganisationType"
+                                       type="radio" value="PrivateUtility" aria-describedby="ca-type-4-item-hint" @IsSelected("PrivateUtility")>
+                            <label class="govuk-label govuk-radios__label" for="@nameof(Model.BuyerOrganisationType)-3">
+                                    @SupplierToBuyerOrganisationTypeModel.BuyerTypes["PrivateUtility"]
+                                </label>
+                                <div id="ca-type-4-item-hint" class="govuk-hint govuk-radios__hint">
+                                    For example, private companies that have been granted a special or exclusive right to provide utility activities to the public. Such as National Grid or British Gas.
+                                </div>
+                            </div>
+                            <div class="govuk-radios__item">
+                            <input class="govuk-radios__input" id="@nameof(Model.BuyerOrganisationType)-4" name="BuyerOrganisationType"
+                                       type="radio" value="Other" data-aria-controls="conditional-ca-type-5" @IsSelected("Other")>
+                            <label class="govuk-label govuk-radios__label" for="@nameof(Model.BuyerOrganisationType)-4">
+                                    Other
+                                </label>
+                            </div>
+
+                            <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-ca-type-5">
+                                <div class="govuk-form-group @(otherHasError ? "govuk-form-group--error" : "")">
+                                    <label class="govuk-label" for="other-buyer-type">
+                                        Enter type
+                                    </label>
+                                    @Html.ValidationMessageFor(m => m.OtherValue, "", new { @class = "govuk-error-message", @id = "other-buyer-type-error" })
+
+                                    <div id="ca-type-5-item-hint" class="govuk-hint">
+                                        For example, the establishment of a utilities dynamic market by a non-contracting authority.
+                                    </div>
+
+                                    <input autocomplete="off" class="govuk-input @(otherHasError ? "govuk-input--error" : "")"
+                                        id="OtherValue"
+                                        name="OtherValue"
+                                       type="text" aria-describedby="@(otherHasError ? "other-buyer-type-error" : "ca-type-5-item-hint")"
```

Comment:
the hint always goes in the aria-describedby, _as well as_ the error if there is one

```suggestion
                                       type="text" aria-describedby="ca-type-5-item-hint @(otherHasError ? "other-buyer-type-error" : "")"
```